### PR TITLE
GH Pages: CNCF Sandbox specific logo in the footer

### PR DESCRIPTION
**Pull request by somaritane**
Add CNCF Sandbox specific logo to k8gb.io website footer

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -40,11 +40,14 @@ <h2 class="project-tagline">{{ page.description | default: site.description | de
...
 
       <footer class="site-footer">
-	<h3 class="project-name">k8gb is a CNCF Sandbox Project
-	</br>
-	<a href="https://www.cncf.io" target="blank">
-	    <img src="https://raw.githubusercontent.com/cncf/artwork/master/other/cncf/horizontal/black/cncf-black.svg" height="120" alt="Cloud Native Computing Foundation">
-	</a></h3>
+        <h3 class="project-name">k8gb is a CNCF Sandbox Project
+          </br>
+          <a href="https://www.cncf.io" target="blank">
```
Comment (kuritka) [inline _layouts/default.html:45]:
```suggestion
          <a href="https://www.cncf.io" target="_blank">
```

**Review by ytsarev — APPROVED** [review]

**Review by kuritka — CHANGES_REQUESTED** [review]

File: _layouts/default.html (hunk 1 above)
Comment (somaritane) [reply _layouts/default.html]: right, although both should work, "_blank" is technically correct

**Comment by ytsarev** [comment]
@somaritane just noticed that DCO fails. Please Sign-off the commit

File: _layouts/default.html [hunk 2]
```diff
@@ -40,11 +40,14 @@ <h2 class="project-tagline">{{ page.description | default: site.description | de
       {% include anchor_headings.html html=content anchorBody="§" %}
 
       <footer class="site-footer">
-	<h3 class="project-name">k8gb is a CNCF Sandbox Project
-	</br>
-	<a href="https://www.cncf.io" target="blank">
-	    <img src="https://raw.githubusercontent.com/cncf/artwork/master/other/cncf/horizontal/black/cncf-black.svg" height="120" alt="Cloud Native Computing Foundation">
-	</a></h3>
+        <h3 class="project-name">k8gb is a CNCF Sandbox Project
+          </br>
```
Comment (kuritka) [inline _layouts/default.html:44]:
```suggestion
          <br>
```

**Review by kuritka — CHANGES_REQUESTED** [review]

**Comment by somaritane** [comment]
> @somaritane just noticed that DCO fails. Please Sign-off the commit

@ytsarev interesting, so when I accept the suggested changes in-place - GH doesn't sign-off automatically.
I'll squash and sign-off.

**Review by kuritka — APPROVED** [review]
lgtm

**Review by k0da — APPROVED** [review]

**Pull request merged**